File: Even_or_odd.java
import java.util.Scanner;

public class Even_or_odd {
    static void Check(int num){
        if(num%2==0){
            System.out.println(num+" is Even.");
        }
        else{
            System.out.println(num+" is Odd.");
        }
    }
    public static void main(String[] args)
    {
        Scanner ip = new Scanner(System.in);
        System.out.println("Enter a Number:");
        int num = ip.nextInt();
        Check(num);
        ip.close();
    }
}

File: Even_Or_Odd.cpp
#include<iostream>
using namespace std;

void Check(int num){
    if(num%2==0)
        cout<<num<<" is Even.\n";
    else
        cout<<num<<" is Odd.\n";
}

int main()
{
    int num;
    cout<<"Enter a Number:";
    cin>>num;
    Check(num);
}